react/ProductGiftName.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC } from 'react'
+import { Link } from 'vtex.render-runtime'
+import { useCssHandles } from 'vtex.css-handles'
+
+import { useGift } from './components/ProductGift'
+
+interface Props {
+  linkToProductPage: boolean
+}
+
+const CSS_HANDLES = ['giftNameLink', 'giftNameText']
+
+const ProductGiftName: FC<Props> = ({ linkToProductPage = true }) => {
+  const handles = useCssHandles(CSS_HANDLES)
+  const gift = useGift()
+
+  return linkToProductPage ? (
+    <Link
+      className={`${handles.giftNameLink} c-on-base link`}
+      href={`/${gift?.linkText}/p`}
```

Comment:
Why checking the existence of gift? Should it be required?

Reply:
Yes! This is not needed anymore